src/diffusers/loaders.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from collections import defaultdict
+from typing import Callable, Dict, Union
+
+import torch
+
+from .models.cross_attention import LoRACrossAttnProcessor
+from .models.modeling_utils import _get_model_file
+from .utils import DIFFUSERS_CACHE, HF_HUB_OFFLINE, logging
+
+
+logger = logging.get_logger(__name__)
+
+
+ATTN_WEIGHT_NAME = "pytorch_attn_procs.bin"
```

Comment:
Actually maybe it would make more sense to make the naming more general here:

```suggestion
ATTN_WEIGHT_NAME = "embeddings.bin"
```

So that multiple loaders could be applied on the same file? cc @pcuenca @patil-suraj

Reply:
I thought about it, but wasn't sure. Another idea would be to make it more specific and descriptive, like `lora_embeddings.bin` and then use different names for others. Not sure what would be easiest to deal with going forward.

Reply:
Not sure if `embeddings` is a good name, since these aren't embeddings, agree with Pedro, maybe make it specific to procs, for example `lora_layers.bin` or `lora_weights.bin`.

Reply:
Actually, instead of hardcoding it - I'll force the user to write the name of the file now instead. Think that's the best/easiest

Reply:
Alright, @patil-suraj convinced me to go for LoRA weights specific name - this means in the longer run:
- We have different files for different parts of the pipeline
- The user will call multiple loading methods for different parts of the pipeline, e.g.:
```python
from diffusers import DiffusionPipeline

pipe = DiffusionPipeline.from_pretrained("...")
pipe.unet.load_attn_procs("...")
pipe.load_text_embeddings("...")
```

But I think that's fine!